src/features/user/Account/PauseModal.tsx

```diff
@@ -150,15 +152,8 @@ export const PauseModal = ({
             {showCalender ? (
               <>
                 <ThemeProvider theme={materialTheme}>
-                  <MuiPickersUtilsProvider
-                    utils={DateFnsUtils}
-                    // locale={
-                    //   localeMapForDate[userLang]
-                    //     ? localeMapForDate[userLang]
-                    //     : localeMapForDate['en']
-                    // }
-                  >
-                    <Calendar
+                  <LocalizationProvider dateAdapter={AdapterDateFns}>
+                    <MuiCalenderPicker
```

Comment:
Selected date colours need to be adjusted. Font colour should be white, and background colour may need to be reviewed.
![Screenshot 2022-05-05 at 5 34 22 PM](https://user-images.githubusercontent.com/44917347/166919429-b45cde39-2be6-48ac-9b95-41d510a81033.png)

Reply:
@mohitb35 MUI-date-pickers comes from [@mui/lab](https://www.npmjs.com/package/@mui/lab).
Components in Mui lab are not completely stable. So for now Mui Core doesn't provide a standard way to style these components.

Reply:
https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1466#discussion_r867484958

Reply:
Issue created #1486